### DISCUSSION TITLE
UnlitMaterial创建的材质动态修改baseTexture，ios手机能够正常显示，部分安卓机无法正常显示，一片黑色

### DISCUSSION THREAD

**Discussion by jasonxs45** [galacean/discussion#101] [Engine] [answered]
如题，本意是使用Texture2D资源作为baseTexture，不同条件下使用不同的图片作为纹理背景，ios手机正常，但是部分安卓手机无法显示动态修改的图片

**Comment by cptbtptpbcptdtptp** [comment]
麻烦提供一下复现的 Demo 与环境（引擎版本，运行在Web端或小程序，浏览器版本，以及部分出现此问题的安卓手机型号）。

**Reply by jasonxs45** [reply]
调整了图片的尺寸，判断是图片尺寸过大造成的，现在问题是，是否有一个标准限制使用图片的尺寸？

**Reply by eyworldwide** [reply]
手机上长或宽尺寸都不要超过 2048

**Reply by zhuxudong** [reply, marked as answer]
出于性能，移动端不要使用超过2048的贴图，此外，我们会在后面版本加上超出限制的警告
